UWPInjector/source/main.cpp

```diff
@@ -70,8 +71,28 @@ void IterateThreads(ThreadCallback ThreadProc, std::uint32_t ProcessID, void* Da
 	CloseHandle(hSnapShot);
 }
 
-int main()
+int main(int argc, char** argv, char** envp)
 {
+	std::uint32_t ProcessID = 0;
+
+	if (true) {
```

Comment:
Don't think we need an always-true conditional here

Reply:
sorry that was left in from debugging it should be
`if (argc > 1)`

Reply:
this has been fixed